### ISSUE TITLE
Integration does not work if a charger or battery is not present

### ISSUE THREAD

**Issue by ryar27** [markgdev/home-assistant_renogy#1]
Works great with a charger and 2 batteries, however once one is removed all stop communicating.  Would it be possible to fix this so it can be used without batteries or charger?  TIA.  Great job on this, BTW.

**Comment by YoloZmolo** [comment]
Anyone was able to make it work only with one component? I tried with correct device ID and MAC but still no success.

**Comment by person65278** [comment]
> Anyone was able to make it work only with one component? I tried with correct device ID and MAC but still no success.

Yes use the same id as your device.. in my case inverter was 32 so i used 32 for the battery also.
